src/components/spinner/spinner.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select } from '@storybook/addon-knobs';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import Spinner from './spinner';
+import notes from './notes.md';
+
+storiesOf('Spinner', module).add('default', () => {
+  const as = select('as', OptionsHelper.colors, Spinner.defaultProps.as);
+  const size = select('size', OptionsHelper.sizesFull, Spinner.defaultProps.size);
+
+  return (
+    <Spinner as={ as } size={ size } />
+  );
+}, { notes: { markdown: notes } });
```

Comment:
You can remove the `@` prop descriptors, as we're not using them. You can keep the description, but get rid of things like `@property as`, `@type {String}` and `@default info`.

![Screenshot 2019-03-13 at 15 11 26](https://user-images.githubusercontent.com/4471057/54290145-584d7300-45a2-11e9-9a86-a1515ec2786d.png)